html/browsers/origin/cross-origin-objects/cross-origin-objects.html

```diff
@@ -256,15 +256,23 @@
 }, "[[DefineOwnProperty]] Should throw for cross-origin objects");
 
 /*
- * [[Enumerate]]
+ * EnumerateObjectProperties (backed by [[OwnPropertyKeys]])
  */
 
 addTest(function() {
-  for (var prop in C)
-    assert_unreached("Shouldn't have been able to enumerate " + prop + " on cross-origin Window");
-  for (var prop in C.location)
-    assert_unreached("Shouldn't have been able to enumerate " + prop + " on cross-origin Location");
-}, "[[Enumerate]] should return an empty iterator");
+  let i = 0;
+  for (var prop in C) {
+    i++;
+    assert_true(prop in whitelistedWindowPropNames, prop + " is not safelisted for a cross-origin Window");
```

Comment:
I think there's a length check following this that tests that aspect.

Good catch on .includes vs. in.